project_cost_report/static/src/js/report_action.js

```diff
@@ -370,6 +372,23 @@ var ReportAction = Widget.extend(ControlPanelMixin, {
             ],
         });
     },
+    /**
+     * Handle a click on the name of a purchase order (POXXXXX)
+     *
+     * The form view of the PO is opened.
+     *
+     * @param {jQuery.Event} - the click event
+     */
+    purchaseOrderNameClicked(event){
+        event.preventDefault();
+        var orderId = getRecordIdFromEvent(event);
+        this.do_action({
+            res_model: "purchase.order",
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Identifier 'res_model' is not in camel case. (camelcase)](https://app.codacy.com/app/numigi/odoo-project-addons/pullRequest?prid=3730559)